server/go/scripts/test.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+#Using https://github.com/gotestyourself/gotestsum for reference
+
+# Looks like CGO has to be enabled for -race if we're using go modules
+CGO_ENABLED=1 gotestsum --junitfile junit_results.xml -- -race -coverprofile coverage.out -v ./...
```

Comment:
`race` is not needed
also is there an extra `--` in the statement?

Reply:
`--` is needed if you want to pass go test flags to gotestsum